src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -31,48 +31,61 @@
 using SonarLint.VisualStudio.Integration.NewConnectedMode;
 using SonarLint.VisualStudio.Integration.Persistence;
 using SonarLint.VisualStudio.Integration.Resources;
+using SonarQube.Client;
 using SonarQube.Client.Models;
 using Language = SonarLint.VisualStudio.Core.Language;
 
 namespace SonarLint.VisualStudio.Integration.Binding
 {
     internal class BindingProcessImpl : IBindingProcess
     {
-        private readonly IHost host;
         private readonly BindCommandArgs bindingArgs;
         private readonly ISolutionBindingOperation solutionBindingOperation;
         private readonly IBindingConfigProvider bindingConfigProvider;
+        private readonly ISonarQubeService sonarQubeService;
+        private readonly IConfigurationPersister configurationPersister;
         private readonly IExclusionSettingsStorage exclusionSettingsStorage;
         private readonly IEnumerable<Language> languagesToBind;
+        private readonly ILogger logger;
 
-        public BindingProcessImpl(IHost host,
```

Comment:
Previously this class used `IHost` to get hold of `ISonarQubeService`, `ILogger` and `IConfigurationPersister`.
It now imports them directly so we can drop `IHost` from this class.

For now, `IConfigurationPersister` is still exposed as an `ILocalService` via `
In the next PR I'll remove it as a service.